core/src/main/java/io/questdb/griffin/engine/functions/groupby/FirstStrGroupByFunction.java

```diff
@@ -73,7 +73,7 @@ public boolean isScalar() {
     @Override
     public void pushValueTypes(ArrayColumnTypes columnTypes) {
         this.valueIndex = columnTypes.getColumnCount();
-        columnTypes.add(ColumnType.INT);
+        columnTypes.add(ColumnType.STRING);
```

Comment:
I guess this should use `ColumnType.BOOLEAN` - this method defines value types stored in the GROUP BY's map. Our map support fixed-size value types only, so it won't be able to store a string value.

Reply:
updated